app/routes/wow.ilvl-export-search.tsx

```
@@ -353,64 +356,91 @@ const IlvlExportSearchComponent = () => {
+                }))
+              }
+            />
+          </div>
           <div className="flex flex-col gap-2">
-            <label className="text-sm font-medium dark:text-gray-200">
-              Desired Stats
-            </label>
+            <div className="flex flex-1 items-center gap-1 relative">
+              <label className="text-sm font-medium dark:text-gray-200">
+                Desired Stats
+              </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)